CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Fix publish script to use npm version
 - Add stage to saucelabs session name for integration tests
+- FIX Audio-video session stop to return Left status code
```

Comment:
- nit: ```FIX Audio-video``` => ```Fix audio-video```
- Could you also include what you have fixed in the commit message instead of ```Fix #93```?